cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringErrorResponse.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.tethering;
+
+/**
+ * Error response for tethering operations sent by the server to the tethering client.
+ */
+public class TetheringErrorResponse {
```

Comment:
on closer examination, it does not look like it makes sense to have this, we should just be using the controlresponse class. Instead, let's use a new TetheringControlStatus enum in the ControlResponse that has PENDING, ACCEPTED, and NOT_FOUND as values (can also temporarily have a REJECTED status until we change the API + UI to delete on rejection).

Reply:
fixed